sgp4/propagation.py

```diff
@@ -1129,8 +1129,8 @@ def _dspace(
 """
 
 def _initl(
-       # not needeed. included in satrec if needed later
-       # satn,
+       # not needeed. included in satrec if needed later 
```

Comment:
Is this extra whitespace intentional, or the result of a merge or something like that? (Also take a look at the several times it happened further down.)

Reply:
Don't know why its there.

Reply:
Roger. I'll try adding `pyflakes` to the project's CI, so that it catches trailing whitespace as part of the build, so that this can be caught automatically next time.

Reply:
I've removed the errant whitespace in an upcoming commit.